tests/regression/7216/test_0001.xml

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE UGENE_TEST_FRAMEWORK_TEST>
+<multi-test>
+<run-cmdline
+        task="!common_data_dir!cmdline/pcr/pcr.uwl"
+        in="!common_data_dir!fasta/pcr_test.fa"
+        primers="!common_data_dir!primer3/not_for_production.lib"
+        out="!tmp_data_dir!7216_out.gb"
+        message="Unexpected symbol: Y"
```

Comment:
Where is the check for the URL?

Reply:
Its impossible to check message with file path because it is dynamic thing.
I changed message, hope you will be satisfied.

Reply:
You can simply add a checked for the regex message (extra param like "messageType"="regex")
But OK, this is optional.